src/index.html

```diff
@@ -4,9 +4,334 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&family=Poppins:wght@400;500;600;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+        src="images/logo.png"
+        alt="moyo-img"
+        class="moyo-img"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#apple" class="nav__link is-active">Apple</a></div></li>
+          <li class="nav__item"><a href="#samsung" class="nav__link">Samsung</a></li>
+          <li class="nav__item"><a href="#smartphones" class="nav__link">Smartphones</a></li>
+          <li class="nav__item">
+            <a
+              href="#laptops-computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item"><a href="#gadgets" class="nav__link">Gadgets</a></li>
+          <li class="nav__item"><a href="#tablets" class="nav__link">Tablets</a></li>
+          <li class="nav__item"><a href="#photo" class="nav__link">Photo</a></li>
+          <li class="nav__item"><a href="#video" class="nav__link">Video</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <section class="cards">
+        <div class="card" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+            class="card__img"
+          >
+          <p class="card__text card__text--y">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+          <p class="card__code">
+            Product code: 195434
+          </p>
+
+          <div class="stars stars--y">
```

Comment:
Fix all cases. Star modifier should work 
![image](https://github.com/mate-academy/layout_catalog/assets/70009205/de94142a-cabe-407f-a897-db36cdd33d93)


```suggestion
          <div class="stars stars--4">
```